app/app.go

```
@@ -51,6 +67,14 @@
 		}
 		log.Info("Bridge %d created", i)
 
+        if b.Direction == "up" {
+            mqtt.Subscribe(newBridge.Topic(), newBridge.IncomingPktHandler, newBridge.BridgeID())
```

golangci-lint (1.62.2)

Error return value of `mqtt.Subscribe` is not checked

(errcheck)